_source/user-guide/integrations/endpoints.md

```diff
@@ -16,6 +16,9 @@ or when Logz.io finds new insights in your logs.
 Logz.io supports sending notifications to
 BigPanda, Datadog, PagerDuty, Slack, VictorOps, or your own custom endpoint.
 
+Alerts are sent on ports 80 and 443 only; setting a custom port as part of endpoint URL will result in alerts not being sent to said endpoint.
```

Comment:
Couple questions here - 

1. Seems to me the key info here is "don't use a custom port", right? If that's the case, can we simplify a little and leave the port numbers out?
2. Does this apply to alerts only, or does it include all notifications?

Reply:
@boofinka

Reply:
1. In my experience, saying "don't use custom ports" invites the question "Well, what ports can be used?". We're going to answer these questions anyway when they come to support, so I'm not sure what we gain from not being transparent.

1. dsf